tox.ini

```diff
@@ -72,3 +72,4 @@ commands=
     py.test --cov-report= --cov=. -v -s -k test_debug_timing --debug-tests -
     python {toxinidir}/run_coveralls.py
     codecov -e TOXENV
+    pip install -r requirements/tests.txt
```

Comment:
i wasnt sure what the problem is (still not 100%)

i believe its basically that requirements parser in setup.py is installing the wrong pytest-django version (and maybe other stuff)

also, its installing to different directories because its being triggered by make/setup.py and not by pip